packages/webapp/public/locales/en/common.json

```diff
@@ -55,6 +59,8 @@
   "SAVE_CHANGES": "Save Changes",
   "SEARCH": "Search",
   "SELECT": "Select",
+  "SELECT_ALL_COUNT": "Select all {{count}}",
```

Comment:
Is "Select all 1" okay or does this need one of those special _one, _other modifiers?

Reply:
What should it be? Should I ask Loïc?

Reply:
I don't have a good idea. "Select 1" doesn't make sense to me either...